app/components/channel_drawer/channels_list/list/list.js

```diff
@@ -218,6 +227,25 @@ class List extends PureComponent {
         this.setState({width: width - 40});
     };
 
+    onPreview = (item) => {
+        if (Platform.OS === 'ios') {
+            const {intl} = this.context;
+
+            this.props.navigator.push({
+                screen: 'ChannelPeek',
+                previewCommit: false,
+                previewView: this.itemRefs[item],
```

Comment:
Do we need this? I can't see it being used anywhere and it's the only place we use the refs as well

Reply:
Yes we do is part of the navigation API for the peek and pop